src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+     data-qa="big"
+     class="search"
+     action="URL"
+     method="post"
+    >
+
+      <label>
+        <img
```

Comment:
```suggestion
    >
      <label>
        <img
```
do not leave empty line between parent and child